README.md

```diff
@@ -52,6 +54,29 @@ If you want to run your Jupyter Notebook on a schedule, you need to be assured t
 
 The solution is Argo-Jupyter-Scheduler: Jupyter-Scheduler front-end with an Argo-Workflows back-end.
 
+## Optional features
+
+### Sending to Slack
+
+Argo-Jupyter-Scheduler allows sending HTML output of an executed notebook to a
+Slack channel:
+
+- See the Slack API docs on how to create a bot token (starts with `xoxb`)
+- Invite your bot to a Slack channel which will be used for sending output
+- When scheduling a notebook (as described above):
+  - Select a conda environment that has `papermill` installed
+  - Add the following `Parameters`:
+    - name: `SLACK_TOKEN`, value: `xoxb-<Slack bot token>`
+    - name: `SLACK_CHANNEL`, value: `<Slack channel name>` (without leading `#`, like `scheduled-jobs`).
+
+Create job:
+
+<img src="./assets/create-job-slack.png" alt="Create job Slack" width="400"/>
+
+Slack output:
+
+<img src="./assets/slack-output.png" alt="Slack output" width="800"/>
```

Comment:
Added docs with screenshots.

Is there a reason the existing screenshots in this README are hosted on GitHub instead of having images in the repo? I'm not sure how long GitHub retains those. We might lose those one day.